form.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require 'functions.php';
+
+$db = new PDO("mysql:host=db; dbname=collection_app", "root", "password");
+$db ->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
Since you're now having to repeat this connection code twice, I'd suggest creating a function that returns a PDO connection just to be a tad DRY-er!